L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp3.cc

```diff
@@ -112,6 +118,8 @@ class HGCalTriggerGeometryV9Imp3 : public HGCalTriggerGeometryBase {
   HGCalGeomRotation::WaferCentring getWaferCentring(unsigned layer, int subdet) const;
   void etaphiMappingFromSector0(int& ieta, int& iphi, unsigned sector) const;
```

Comment:
etaphiMappingFromSector0 is used to rotate the scintillator module, not the trigger cells. `hSc_num_panels_per_sector_` should be 12 now though I think, so I've updated it.

I don't think we need an inverse of `tcEtaphiMappingToSector0` since we get use the `getTriggerCellsFromModule` once we've rotated the module to sector 0.

Reply:
Ok, thanks